src/main/resources/application-aws.properties

```diff
@@ -52,8 +52,8 @@ oicd.publickey=-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBC
 
 #Your domain here:
 application.url=NEEDS_OVERRIDE
-application.name=Unicon LTI 1.3 Demo
-application.description=The Unicon tool to learn about LTI 1.3 in java
+application.name=Lumen Waymaker 1.3
```

Comment:
Sorry, Mary, but could these be even more generic.  Like: Lumen Learning LTI 1.3, Lumen Learning LTI 1.3 integration, etc?

Reply:
Updated!